bats/tests/compose/compose.bats

```diff
@@ -0,0 +1,49 @@
+load '../helpers/load'
+
+local_setup() {
+    if is_windows && using_containerd && using_windows_exe; then
+         # BUG BUG BUG
+         # There is a known issue of nerdctl compose not working as expected in
+         # WSL distros. https://github.com/rancher-sandbox/rancher-desktop/issues/1431
+         skip "Test doesn't work with nerdctl in a WSL distro"
+     fi
+    TESTDATA_DIR="${PATH_BATS_ROOT}/tests/compose/testdata/"
+    TESTDATA_DIR_HOST=$(host_path "$TESTDATA_DIR")
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+skip_on_nerdctl_in_wsl_using_windows_exe() {
+    if is_windows && using_containerd && using_windows_exe; then
+        # There is a known issue of nerdctl compose not working as expected in 
+        # WSL distros. https://github.com/rancher-sandbox/rancher-desktop/issues/1431
+        skip "Test doesn't work with nerdctl in a WSL distro"
+    fi
+}
+
+@test 'start container engine' {
+    skip_on_nerdctl_in_wsl_using_windows_exe
```

Comment:
And neither are any of these calls. That was the point of moving the "skip" logic into `local_setup`.